src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
     children: PropTypes.node,
     className: PropTypes.string,
+    href: PropTypes.string,
     to: PropTypes.string,
-    size: PropTypes.string.isRequired,
   }
 
-  static defaultProps = {
-    appearance: undefined,
```

Comment:
Is there an issue on file to put something like `appearance` back? I would rather see it done in this patch because I think it will be hard to track down all the hard coded class names that are used in component instances. It would be ok to do it in a cleanup patch though.